source/concepts/recording.rst

```diff
@@ -70,7 +139,7 @@ In order to send both streams together, provide the two file paths separated by
 then choose the files.
 
 
-Reading recordings using python
+Reading recordings using mdanalysis in python
```

Comment:
```suggestion
Reading NanoVer recordings using MDAnalysis in python
```
I think it would be good to make it clear throughout this page that we are talking about NanoVer recordings, i.e. recordings made using the inbuilt recording functionality of NanoVer (as it is possible to record trajectories in other ways)

Reply:
Done in commit [52dc633](https://github.com/IRL2/nanover-docs/pull/113/commits/52dc633153e8d43308b50314cf888181b9394a23).